restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnection.java

```diff
@@ -102,6 +102,14 @@
         DISCONNECTED,  /** Connection is in state disconnected */
     };
 
+    /**
+     * General status.
+     */
+    public enum Status {
```

Comment:
I don't really like to use bool for things which are not really true / false, yes / no, where it's not obvious from the method name what true and false means.  So for methods like haveConnectivity() it would be ok, because true means we have connectivity, false means we don't.  But for setupWebrtcAndCall() I would not be sure what true means.
But it's just me, you can change it if you want :)